frontend/apps/filemanager/filemanagermenu.lua

```diff
@@ -188,13 +188,14 @@ function FileManagerMenu:setUpdateItemTable()
     table.insert(self.tab_item_table.tools, {
         text = _("OPDS catalog"),
         callback = function()
-            local FileManager = require("apps/filemanager/filemanager")
             local OPDSCatalog = require("apps/opdscatalog/opdscatalog")
-            function OPDSCatalog:onExit()
-                FileManager:onRefresh()
-            end
-            OPDSCatalog:showCatalog()
-        end,
+            local callback_filemanager_refresh = function() self.ui.callback_refresh() end
```

Comment:
Why not just call `self.ui:onRefresh` directly here? That way you don't have to create the callback closure in filemanager.

Also please to keep the function name naming convention consistent as @Frenzie mentioned ;)